### DISCUSSION TITLE
Display file absolute path

### DISCUSSION THREAD

**Discussion by 4goodapp** [coteditor/CotEditor#1346] [Q&A] [answered]
Is there way to display the whole file path instead of just the file name?

**Comment by 1024jp** [comment, marked as answer]
CotEditor doesn't have a feature to change the display of the title bar, but you can find the full path in the document inspector. 

<img width="1017" alt="Screen Shot 2022-08-17 at 17 36 08" src="https://user-images.githubusercontent.com/1165044/185073880-6bfefb98-980d-4942-9abd-caf3f1b3e824.png">

**Reply by 4goodapp** [reply]
Thanks 😊 🤩 .
